app/dcs/api/s3/s3-compatibility/page.md

```diff
@@ -120,6 +120,19 @@ Full compatibility means that we support all features of a specific action
 except for features that rely on other actions that we haven't fully
 implemented.
 
+### GetBucketLocation
+
+This is currently supported in Gateway-MT only.
+
+#### LocationConstraint
+
+Specifies the placement region where the bucket data is stored:
```

Comment:
```suggestion
Specifies the placement region where the object data for a bucket is stored:
```

Reply:
Should we specific OBJECT data? Given that this is grabbed from the satellite, this is more accurate right?